test/MOI_wrapper.jl

```diff
@@ -3,8 +3,12 @@ import HiGHS
 
 const MOI = MathOptInterface
 
-@testset "SolverName" begin
-    @test MOI.get(HiGHS.Optimizer(), MOI.SolverName()) == "HiGHS"
+@testset "Attributes" begin
+    o = HiGHS.Optimizer()
+    @test MOI.get(o, MOI.SolverName()) == "HiGHS"
+    @test MOI.get(o, MOI.TimeLimitSec()) > 10000
+    MOI.set(o, MOI.TimeLimitSec(), 500)
+    @test MOI.get(o, MOI.TimeLimitSec()) ≈ 500
```

Comment:
There are no arithmetic operations involved, so it should be safe to check for exact equality.

Reply:
I wasn't sure there wasn't some operations done on the highs side, but yes it makes sense, fixed in last commit